gr-blocks/lib/test_tag_variable_rate_ff_impl.cc

```diff
@@ -82,11 +80,7 @@ namespace gr {
       GR_LOG_DEBUG(d_logger, boost::format("noutput_items: %1%") % noutput_items);
 
       if(d_update_once) {
-#ifdef HAVE_RAND48
-        if(drand48() > 0.5) {
-#else
-        if (rand() > RAND_MAX / 2) {
-#endif
+        if (xoroshiro128p_next(&d_state[0]) > (UINT64_MAX / 2)){
```

Comment:
1. why `&d_state[0]`? `type[size]` gets automatically converted to `*type` on calling a C-Calling-Convention function. 
2. don't do a comparison with "half-scale" to get a bit of output. Take an arbitrary but the lowest and highest bit and use that as bit.